editor/components/block-list/block.js

```diff
@@ -439,7 +439,7 @@ export class BlockListBlock extends Component {
 		// Insertion point can only be made visible when the side inserter is
 		// not present, and either the block is at the extent of a selection or
 		// is the last block in the top-level list rendering.
-		const shouldShowInsertionPoint = (
+		const shouldShowInsertionPoint = ! isLocked && (
```

Comment:
For the block movers, we assign `shouldShowMovers` without consideration of `isLocked`, and this consideration is managed by the `BlockMover` component itself. Should we do that here? Or at least align the two approaches one way or the other?

Reply:
Yes, we should be consistent, I updated the code to follow the same logic we now do the check for template locking inside BlockInsertionPointm, like we do for BlockMover.